internal/config/config.go

```diff
@@ -114,6 +116,14 @@ func extractVersionAnalyzersArguments(cmd *cobra.Command) {
 				logrus.Errorf("couldn't add %v flag: %v", argNamePrefix+argDefinition.Name, err.Error())
 				continue
 			}
+
+			envKey := strings.ToUpper(argNamePrefix + argDefinition.Name)
+			envKey = strings.ReplaceAll(envKey, "-", "_")
+			envKey = strings.ReplaceAll(envKey, ".", "_")
```

Comment:
That's really nice! but for the sake of consistency, please also update the other env variables (like `SEMANGIT_LOGLEVEL`) or remove these chars at all.